src/storage/src/hummock/store/local_hummock_storage.rs

```diff
@@ -337,6 +343,26 @@ impl LocalStateStore for LocalHummockStorage {
         .await
     }
 
+    async fn try_flush(&mut self) -> StorageResult<()> {
+        if self.mem_table.kv_size.size() > self.mem_table_spill_threshold {
+            tracing::info!(
+                "The size of mem table exceeds {} Mb and spill occurs. table_id {}",
+                self.mem_table_spill_threshold >> 20,
+                self.table_id.table_id()
+            );
+
+            if self.spill_offset < MAX_SPILL_TIMES {
+                self.spill_offset += 1;
```

Comment:
Better to increment the spill_offset in flush? Otherwise the flush trigger by barrier will have the same spill_offset as the previous spill.